### PR TITLE
Alicefate/ add variable to FormField and OpenResponse

### PR DESCRIPTION
**Pull request by alicefate**
add value to this two component, so it could set default value. 
It is helpful when user edit the exsiting idea, they can have their original input.

### CODE REVIEW

**Comment by alicefate** [comment]
I only changed 4 files. FormField.tsx and index.ts in FormField, OpenResponse.tsx and index.ts in OpenResponse. 
Others are add by systom automaticly.

**Review by Enjoy2Live — CHANGES_REQUESTED** [review]
@alicefate looks great, I like these small changes because more of them can lead to generalized props.

left you a question in the comments please answer it and consider this merged :D

File: packages/UI/src/components/organisms/FormField/FormField.tsx [hunk 1]
```diff
@@ -20,12 +20,13 @@ function FormField({
   touched,
   onChange,
   name,
+  value,
 }: InputProps) {
   return (
     <>
       <Label htmlFor={id}>
         <Typography type="label">
-          {label} {required && <span css={{ color: 'red' }}>*</span>}
```
[the remark this answers is not in the document]
Comment (alicefate) [reply packages/UI/src/components/organisms/FormField/FormField.tsx:28]: Compiler can understand better. and it will stop problem warnning.

**Comment by dbradham** [comment]
@Enjoy2Live can you let Iris know if any more changes are need here or approve?

File: packages/UI/src/components/organisms/FormField/FormField.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dbradham) [reply packages/UI/src/components/organisms/FormField/FormField.tsx:28]: @alicefate can you revert this change? It seems to be the only pending item for this PR

[the remark this answers is not in the document]
Comment (alicefate) [reply packages/UI/src/components/organisms/FormField/FormField.tsx]: sure

**Review by Enjoy2Live — APPROVED** [review]
LGTM!

**Pull request merged**